test/integration/crossPartition.spec.ts

```diff
@@ -340,6 +341,34 @@ describe("Cross Partition", function() {
       await executeQueryAndValidateResults({ query, options, expectedOrderIds: expectedOrderedIds });
     });
 
+    it("Validate DISTINCT Query", async function() {
+      // simple order by query in string format
+      const query = "SELECT DISTINCT VALUE r.spam3 FROM root r";
+      const options = {
+        enableCrossPartitionQuery: true,
+        maxItemCount: 2
+      };
+
+      const expectedOrderedIds = Array.from(new Set(documentDefinitions.map(r => r["spam3"])));
```

Comment:
Is `Set` ordered? If not then the order is undefined and this test will pass by chance.

Reply:
Removed `Set` here